nevergrad/optimization/rescaledoneshot.py

```diff
@@ -46,33 +46,31 @@
     scale=0.01, sampler="Hammersley", scrambled=True).with_name("Recentering0ScrHammersleySearch", register=True)
 Recentering0ScrHaltonSearch = SamplingSearch(
     scale=0.01, sampler="Halton", scrambled=True).with_name("Recentering0ScrHaltonSearch", register=True)
-ORecentering1ScrHammersleySearch = SamplingSearch(quasi_opposite="opposite",
-    scale=0.1, sampler="Hammersley", scrambled=True).with_name("ORecentering1ScrHammersleySearch", register=True)
-ORecentering4ScrHammersleySearch = SamplingSearch(quasi_opposite="opposite",
-    scale=0.4, sampler="Hammersley", scrambled=True).with_name("ORecentering4ScrHammersleySearch", register=True)
-QORecentering4ScrHammersleySearch = SamplingSearch(quasi_opposite="quasi",
-    scale=0.4, sampler="Hammersley", scrambled=True).with_name("QORecentering4ScrHammersleySearch", register=True)
-ORecentering1ScrHaltonSearch = SamplingSearch(quasi_opposite="opposite",
-    scale=0.1, sampler="Halton", scrambled=True).with_name("ORecentering1ScrHaltonSearch", register=True)
-ORecentering4ScrHaltonSearch = SamplingSearch(quasi_opposite="opposite",
-    scale=0.4, sampler="Halton", scrambled=True).with_name("ORecentering4ScrHaltonSearch", register=True)
-ORecentering7ScrHammersleySearch = SamplingSearch(quasi_opposite="opposite",
-    scale=0.7, sampler="Hammersley", scrambled=True).with_name("ORecentering7ScrHammersleySearch", register=True)
-QORecentering7ScrHammersleySearch = SamplingSearch(quasi_opposite="quasi",
-    scale=0.7, sampler="Hammersley", scrambled=True).with_name("QORecentering7ScrHammersleySearch", register=True)
-ORecentering20ScrHaltonSearch = SamplingSearch(quasi_opposite="opposite",
-    scale=2.0, sampler="Halton", scrambled=True).with_name("ORecentering20ScrHaltonSearch", register=True)
-ORecentering20ScrHammersleySearch = SamplingSearch(quasi_opposite="opposite",
-    scale=2.0, sampler="Hammersley", scrambled=True).with_name("ORecentering20ScrHammersleySearch", register=True)
-ORecentering12ScrHaltonSearch = SamplingSearch(quasi_opposite="opposite",
-    scale=1.2, sampler="Halton", scrambled=True).with_name("ORecentering12ScrHaltonSearch", register=True)
-ORecentering12ScrHammersleySearch = SamplingSearch(quasi_opposite="opposite",
-    scale=1.2, sampler="Hammersley", scrambled=True).with_name("ORecentering12ScrHammersleySearch", register=True)
-ORecentering7ScrHaltonSearch = SamplingSearch(quasi_opposite="opposite",
-    scale=0.7, sampler="Halton", scrambled=True).with_name("ORecentering7ScrHaltonSearch", register=True)
-ORecentering0ScrHammersleySearch = SamplingSearch(quasi_opposite="opposite",
-    scale=0.01, sampler="Hammersley", scrambled=True).with_name("ORecentering0ScrHammersleySearch", register=True)
-ORecentering0ScrHaltonSearch = SamplingSearch(quasi_opposite="opposite",
-    scale=0.01, sampler="Halton", scrambled=True).with_name("ORecentering0ScrHaltonSearch", register=True)
-
-
+ORecentering1ScrHammersleySearch = SamplingSearch(opposition_mode="opposite",
```

Comment:
I still feel that registering all this is unnecessary, I'll try to convince you again eventually :p

Reply:
No pb we remove registering :-)  except for the autorescale maybe.

Reply:
There is no registered autorescale :s I let you decide what you prefer on the actual PR (I merged this one in)